lib/java_buildpack/framework/dynatrace_one_agent.rb

```diff
@@ -126,8 +135,8 @@ def agent_manifest
 
       def agent_path(manifest)
         technologies  = manifest['technologies']
-        java_binaries = technologies['java']['linux-x86-64']
-        loader        = java_binaries.find { |bin| bin['binarytype'] == 'loader' }
+        java_binaries = technologies['process']['linux-x86-64']
```

Comment:
why change from `java` to `process` ?

Reply:
The use of `java` is for historical reasons. `process` is not the default and also used in other buildpacks (e.g. https://github.com/Dynatrace/libbuildpack-dynatrace/blob/master/hook.go#L403)

Reply:
ok!